src/app/@ansyn/map-facade/components/annotation-context-menu/annotation-context-menu.component.html

```diff
@@ -4,8 +4,9 @@
 	 [style.width.px]="clickMenuProps .boundingRect.width"
 	 [style.height.px]="clickMenuProps.boundingRect.height"
 	 [style.left.px]="clickMenuProps.boundingRect.left"
-	 eventListener="pointerdown"
+	 [clickEventType]="['pointerdown', 'mousedown']"
```

Comment:
why not 'mousedown' only ?
what is `clickEventType` ? maybe `eventListeners`?

Reply:
`mousedown` does not work for current Chrome (but it works for Chrome 44).
`pointerdown` does not work for Chrome 44 (but it works for current Chrome).

The word "listener" usually refers to a function. Here we have not functions, but strings, which denote the _event type(s)_: click, mousedown, etc.